Tests that console.error does not throw exception when executed in console on call frame.
